packages/cmrjs/cmr-utils.js

```diff
@@ -352,7 +352,8 @@ function constructOnlineAccessUrl({
   buckets
 }) {
   const bucketType = buckets.type(file.bucket);
-  if (bucketType === 'protected') {
+  const urlBuckets = ['protected', 'public'];
```

Comment:
Rename `urlBuckets` to `distributionApiBuckets`? It seems like it might be clearer since the variable is used to determine whether the files in a bucket should be protected by the distribution API

Reply:
that way better.